tiledb/api/c_api/vfs/vfs_api_experimental.h

```diff
@@ -0,0 +1,101 @@
+/**
+ * @file tiledb/api/c_api/vfs/vfs_api_experimental.h
+ *
+ * @section LICENSE
+ *
+ * The MIT License
+ *
+ * @copyright Copyright (c) 2023 TileDB, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ *
+ * @section DESCRIPTION
+ *
+ * This file declares the experimental VFS C API for TileDB.
+ */
+
+#ifndef TILEDB_VFS_API_EXPERIMENTAL_H
+#define TILEDB_VFS_API_EXPERIMENTAL_H
+
+#include "tiledb/api/c_api/api_external_common.h"
+#include "tiledb/api/c_api/context/context_api_external.h"
+#include "tiledb/api/c_api/vfs/vfs_api_external.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * Typedef for ls_recursive callback function invoked on each object collected.
+ *
+ * @param path The path of a visited object for the relative filesystem.
+ * @param path_len The length of the path.
+ * @param object_size The size of the object at the current path.
+ * @param data Data passed to the callback used to store collected results.
+ */
+typedef int32_t (*LsCallback)(
```

Comment:
```suggestion
typedef int32_t (*tiledb_ls_callback_t)(
```
This is a definition in an external global namespace, so it needs the standard prefix.

Reply:
done